src/cryptoadvance/specter/key.py

```diff
@@ -165,6 +165,14 @@ def json(self):
     def purpose(self):
         return purposes[self.key_type]
 
+    def __str__(self):
+        if self.derivation and self.fingerprint:
+            return f"[{self.fingerprint}/{self.derivation[2:]}]{self.original}"
+        elif self.derivation:
```

Comment:
I think it should be `elif self.fingerprint:` right?

Reply:
ups...